### PR TITLE
Add english story about trapezoid

### PR DESCRIPTION
**Pull request by hychoi99**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1995)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/trapezoid/trapezoid.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+You draw a perfect trapezoid on the blackboard
+
+The finished trapezoid starts to glow and a hole appears
+
+Mystified, you enter the trapezoid
+
+Do you:
+
+[Lunge for the lighter](lunge-for-the-lighter/lunge-for-the-lighter.md)
```
Comment (casey-collab) [inline english/trapezoid/trapezoid.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/trapezoid/trapezoid.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+[Wake up!](wake-up/wake-up.md)
```
Comment (casey-collab) [inline english/trapezoid/trapezoid.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by hychoi99** [comment]
Fixed the broken links, and there doesn't seem to be a merge conflict.

**Comment by cbuckey-uda** [comment]
@hychoi99, thanks for fixing the links! However, I do see a merge conflict. Do you see how at the bottom of the pull request, right above the comment box, it says "This branch has conflicts that must be resolved"? Or is that not showing up for you?

**Comment by hychoi99** [comment]
Hi @cbuckey-uda , I do see the box that says "This branch has conflicts that must be resolved." Does this mean I have a merge conflict between this branch and the master branch from udacity I'm pull requesting with?

Using git, I tried to merge this branch with the master branch from my fork from but no merge conflicts are shown. Since my master branch is essentially a clone of the master branch from udacity, I'm not sure where the conflict is. My changes were only additions of lines!

**Comment by cbuckey-uda** [comment]
@hychoi99, yes, that means there are conflicts between your branch and the master branch from udacity.

The problem with merging master from your fork is that the master on the original may be ahead of your fork - your fork will not automatically stay up-to-date. [This video](https://www.udacity.com/course/viewer#!/c-ud775/l-3105028581/e-3306849568/m-3315478816) contains some ideas for how to deal with this, and the solution video has more detail.

As for why there are conflicts, I'm guessing that someone else added lines to `marshmallow.md`.

**Comment by hychoi99** [comment]
Great! I think I resolved the merge conflict.

**Comment by cbuckey-uda** [comment]
I'm glad to hear it! @casey-collab, please take another look.

**Comment by casey-collab** [comment]
Looks great! :dart:

**Pull request merged**